src/main/java/tds/common/data/mapping/ResultSetMapperUtility.java

```diff
@@ -29,10 +32,36 @@ public static Instant mapTimeStampToInstant(ResultSet rs, String columnLabel) th
      * @param time The {@link java.time.Instant} to convert
      * @return A {@link java.sql.Timestamp} representation of the {@link java.time.Instant}
      */
-    public static Timestamp mapInstantToTimestamp(Instant time) {
+    public static Timestamp mapJodaInstantToTimestamp(Instant time) {
         return time == null
             ? null
             : Timestamp.valueOf(LocalDateTime.ofInstant(time, ZoneOffset.UTC));
     }
 
+    /**
+     * Map a {@link org.joda.time.Instant} to a {@link java.sql.Timestamp}
+     *
+     * @param time the {@link org.joda.time.Instant} to convert
+     * @return the converted instant
+     */
+    public static Timestamp mapInstantToTimestamp(org.joda.time.Instant time) {
+        return time == null
+            ? null
+            : new Timestamp(time.getMillis());
+    }
+
+    /**
+     * Map a {@link java.sql.Timestamp} to a {@link org.joda.time.Instant}
+     *
+     * @param rs          {@link java.sql.ResultSet} containing the data to convert
+     * @param columnLabel the column label used to fetch the timestamp from the result set
+     * @return converted {@link org.joda.time.Instant}
+     * @throws SQLException in the event an error occurs accessing the result set
+     */
+    public static org.joda.time.Instant mapTimeStampToJodaInstant(ResultSet rs, String columnLabel) throws SQLException {
```

Comment:
Casing of TimeStamp is inconsistent between methods.  I would vote for Timestamp instead of TimeStamp since that is the casing of the java object.